src/EachPromise.php

```diff
@@ -28,7 +28,7 @@ class EachPromise implements PromisorInterface
     /** @var callable|null */
     private $onRejected;
 
-    /** @var Promise|null */
+    /** @var PromiseInterface|null */
```

Comment:
@GrahamCampbell updated this after seeing https://github.com/guzzle/promises/actions/runs/13716130800/job/38700847561